src/services/utils.js

```diff
@@ -99,6 +99,27 @@ export function loadScript({ url = '', first = false, head = false }) {
   });
 }
 
+// Returns list of random integers
+export const generateUniqueNumbers = (amount, min, max) => {
+  const indexes = [];
+  // sanity check to prevent infinit loop
+  if (amount > (max - min)) {
+    return indexes;
+  }
+  // max tries are arbitrary selected attempts limit
+  // to prevent super expensive cases of finding all
+  // unique numbers
+  let maxTries = 10 * amount;
+  while (indexes.length < amount && maxTries > 0) {
+    maxTries -= 1;
+    const randomInt = getRandomInt(min, max);
+    /* eslint-disable no-continue  */
+    if (indexes.indexOf(randomInt) > -1) continue;
```

Comment:
We can avoid disabling eslint here:
```js
if (!indexes.includes(randomInt)) {
  indexes.push(randomInt);
}
```

(I know it's from the old implementation 😅 )

Reply:
Yes! I realized after pushing, this should be refactored. :) Will updated. :+1: